test/controllers/products_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
This and the other files inside of the `test` directory are the result of creating this rails project with `rails new .` rather than `rails new . -T` (the `-T` option says to not use the default test framework).

Since we'll be using RSpec instead of the default framework, creating our rails apps with `-T` is going to be required for all projects in the future.